include/nng/supplemental/nanolib/conf.h

```diff
@@ -197,7 +197,8 @@ typedef struct {
 	char *   topic;
 	uint32_t topic_len;
 	uint8_t  qos;
-} subscribe;
+	uint32_t stream_id;	    // only effective when multi_stream is enabled
+} topics;
```

Comment:
This structure name is easy to misunderstand.

Reply:
well , gonna use this for publish topic-stream pair